Microsoft.Toolkit.Uwp.Services/Services/Bing/BingDataProvider.cs

```diff
@@ -41,7 +42,7 @@ public class BingDataProvider : DataProviderBase<BingSearchConfig, BingResult>
         protected override async Task<IEnumerable<TSchema>> GetDataAsync<TSchema>(BingSearchConfig config, int maxRecords, IParser<TSchema> parser)
         {
             var countryValue = config.Country.GetStringValue();
-            var locParameter = string.IsNullOrEmpty(countryValue) ? string.Empty : $"loc:{countryValue}+";
+            var locParameter = string.IsNullOrEmpty(countryValue) ? $"loc:{CultureInfo.CurrentCulture.Name.Split('-')[1].ToLower()}+" : $"loc:{countryValue}+";
```

Comment:
Just checked again and found the issue again.

My CurrentCulture is en-US and TwoLetterISOLanguageName is "en". But if you pass "en" to the bing service it will not return any result. It looks like bing is accepting the second part of the culture.

Also[, this documentation](https://msdn.microsoft.com/en-us/library/system.globalization.cultureinfo.twoletterisolanguagename(v=vs.110).aspx) tells that there might be different countries who has the same TwoLetterISOLanguageName.